lib/Transforms/BufferPlacement/BufferingSupport.cpp

```diff
@@ -112,17 +112,6 @@ LogicalResult dynamatic::buffer::mapChannelsToProperties(
   // properties into the channel map. Fails and marks the MILP unsatisfiable if
   // any of those combined buffering properties become unsatisfiable.
   auto deriveBufferingProperties = [&](Channel &channel) -> LogicalResult {
-    ChannelBufProps ogProps = *channel.props;
-    if (!ogProps.isSatisfiable()) {
```

Comment:
I think this if statement is identical to the one below (line 127), isn't it? @QinYuan2000